apps/glusterfs/app_volume_test.go

```diff
@@ -166,12 +166,20 @@ func TestVolumeCreateInvalidSize(t *testing.T) {
 }
 
 func TestVolumeCreateSmallSize(t *testing.T) {
-	tmpfile := tests.Tempfile()
-	defer os.Remove(tmpfile)
 
-	// Create the app
-	app := NewTestApp(tmpfile)
-	defer app.Close()
+	dbfile := tests.Tempfile()
+	defer os.Remove(dbfile)
+	os.Setenv("HEKETI_EXECUTOR", "mock")
+	defer os.Unsetenv("HEKETI_EXECUTOR")
+
+	data := []byte(`{
+		"glusterfs" : {
+			"db" : "` + dbfile + `",
+			"brick_min_size_gb" : 2
```

Comment:
Suggestion: you can use 4 here (instead of 2). Then you don't need to change the test below, hence reducing the size of the patch! :-)